source/support/support-overview.rst

```diff
@@ -0,0 +1,161 @@
+=======
+Support
+=======
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+MinIO users have two options for support.
+
+#. Community support from the `public Slack channel <https://minio.slack.com>`_.
+   
+   Community support is best-effort only and has no :abbr:`SLA (Service Level Agreement)` or :abbr:`SLO (Service Level Objective)`.
+#. A paid commercial license grants access to the MinIO Subscription Network, `SUBNET <https://subnet.min.io>`_ with a 24 hour or 1 hour :abbr:`SLA (Service Level Agreement)` depending on subscription level.
+   
+   For current licensing levels and pricing, refer to the |SUBNET| page.
+
+
+.. _minio-docs-subnet:
+
+SUBNET
+------
+
+MinIO `SUBNET <https://subnet.min.io?ref=docs>`_ delivers 24/7/365 Direct-to-engineer support through a MinIO-built portal that blends the best of Slack and Zendesk.
+
+Features of SUBNET include:
+
+- Security and architecture reviews (depending on :abbr:`SLA (Service Level Agreement)`)
+- Access to Panic Button (depending on :abbr:`SLA (Service Level Agreement)`)
+- Secure communication channel to exchange logs and software binaries
+- Unlimited seats for your team
+- Unlimited issues
+
+
+Registering Your MinIO Deployment with SUBNET
+---------------------------------------------
+
+.. tab-set::
+
+   .. tab-item:: Console
+
+      You can register for SUBNET from the MinIO Console.
+
+      #. Go to your MinIO cluster's URL, then sign in
+      #. Select the :guilabel:`Support` option
+      #. Select :guilabel:`Register`
+      #. Enter your :guilabel:`Email` address and a unique :guilabel:`Password`, then select :guilabel:`Register`
+
+      For clusters without direct Internet access:
+
+      #. Go to your MinIO cluster's URL, then sign in
+      #. Select the :guilabel:`Support` option
+      #. Select :guilabel:`Register`
+      #. Select the :guilabel:`Offline Activation` link at the bottom of the screen
+      #. Copy the registration token for your cluster from item 1
```

Comment:
Maybe we can labelize this section and point to it for `mc support register --airgap`

Reply:
It's Console, not CLI.
I'll see about adding something to the Console docs.

Reply:
Just added a new tab instead. Console docs don't have any procedures yet, and I'm not ready to open that can of worms.